Logger/Logger.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using System.Data;
+using Microsoft.Data.Sqlite;
+
+namespace Logger
+{
+    public class Logger
+    {
+        private const string nameBD = "Data Source=Log.sqlite";
+        private SqliteConnection bd;
```

Comment:
```suggestion
        private SqliteConnection _db;
```
Правильно по англ. БД пишется DB